test/treat-locks/pnpm.lock.test.js

```diff
@@ -0,0 +1,118 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.argv._
+
+const fs = require('fs');
```

Comment:
I'm missing more pnpm-lock-yaml file cases (like the one you have for `actual-file-test/pnpm-lock.yaml`), I am ok to test even the obvious cases like the ones not containing registries at all, or like the one containing registry information but the dependencies not specifying registry (like `registry.npmjs.org/@babel/runtime/7.16.7:` at all)
Sometimes the most stupid cases are the failure ones and we have to cover them, not because your code could fail but a refactoring.

Reply:
Should I get more example pnpm-lock.yaml files and them to the test cases?

Reply:
yes please @shubhbapna , that's my proposal, to get more different pnpm-lock-yaml cases (again, different scenarios) and then compare the current status vs the expected status after running locktt. Do you agree? is it clear?

Reply:
I think all the obvious cases might already be covered since we treating the file line by line.
So for the cases which contains no registries at all would be equivalent to treating a line with no registry which is already covered in the current test cases. Similarly second case is also covered in the actual-file-test.
What do you think?